development_environment

```diff
@@ -0,0 +1,124 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+## FlowAuth settings
+# Database that stores users and servers, defaults to temporary sqlite
+#SQLALCHEMY_DATABASE_URI 
+# Flask secret key for CSRF protection
+SECRET_KEY=secret 
+# At rest encryption of server secret keys
+FERNET_KEY="XU-J5xNOtkaUKAoqWT7_VoT3zk2OTuoqKPBN3l0pOFg=" 
+# Creates demo data when running
+DEMO_MODE=true
+# Required by flask
+FLASK_APP=flowauth 
+# Enable/disable flask debug/autoreload
+FLASK_DEBUG=1
+FLOWAUTH_PORT=9091
+
+## FlowAPI settings
+# JWT secret key
+JWT_SECRET_KEY=secret 
+# Shouldn't be relevant as serving now done by hypercorn?
+QUART_APP="flowapi.main:create_app()" 
+# Shouldn't be relevant as serving now done by hypercorn?
+QUART_DEBUG=1 
+# Error & debug loc level
+LOG_LEVEL=debug 
+# Flowmachine ZMQ host
+FLOWMACHINE_SERVER=localhost
+FLOWAPI_PORT=9090
+
+
+# Hostname to connect to flowdb
+FLOWDB_HOST=localhost 
+# Port to connect to flowdb
+FLOWDB_PORT=9000 
+
+## FlowMachine settings
+# To avoid mpl errors
+MPLBACKEND="agg" 
+# Password for redis
+REDIS_PASSWORD=fm_redis 
+# Error and debugging lo level
+LOG_LEVEL=debug 
+# Flowmachine db user
+FLOWMACHINE_FLOWDB_USER=analyst 
+# Flowmachine db password
+FLOWMACHINE_FLOWDB_PASS=foo 
```

Comment:
`FLOWMACHINE_FLOWDB_USER` and `FLOWMACHINE_FLOWDB_PASS` are also set below in the "FlowDB Settings" section